### PR TITLE
py3: fix stringio imports

### PR DESCRIPTION
**Pull request by jarondl**

### CODE REVIEW

File: misc/import_ch_zurich.py [hunk 1]
```diff
@@ -22,7 +22,10 @@
 # motivation to port it. Please see the examples directory for better
 # examples.
 
-import cStringIO
+try:  #py2
+  from StringIO import StringIO
+except ImportError:
+  from io import StringIO
```
Comment (avilaton) [inline misc/import_ch_zurich.py]: I suggest making a compat.py module and moving this bit in there since this logic is repeated a few times.

Comment (jarondl) [reply misc/import_ch_zurich.py]: That is a good idea.
Where do you suggest to put it? All of the other places are either inside transitfeed or import it. But this specific one, in misc, does not import transitfeed, and it feels like a bad idea to introduce another import here.
So perhaps have a compat inside transitfeed, for all other cases, leaving this one alone? wdyt?

Comment (avilaton) [reply misc/import_ch_zurich.py]: I would focus on working inside `transitfeed` first and having it work on python 3 passing tests. Then this misc would come later

Comment (jarondl) [reply misc/import_ch_zurich.py]: Done

**Comment by avilaton** [comment]
I sent PRs using `six` to see if anyone looks at them.

**Pull request merged**